### DISCUSSION TITLE
A Matrix XApp

### DISCUSSION THREAD

**Discussion by Secret-chest** [linuxmint/discussions#578] [Ideas]
Element is the only WebApp we ship, would be really nice if we replaced it with something native (but that would also function with all rooms)

**Comment by Jopp-gh** [comment]
Well, everyone is free to install the apps they prefer.
 
I've used the most popular chat apps, (WhatsApp, Skype, signal, element, discord, revolt, zoom, etc) they all are are nicely polished in features and aesthetics, but not "private" at all, everything is tracked. The buzzword "privacy" is just a myth. In absence of privacy I prefer to have none of these apps on my Linux machine. Alternatives? Well, I still need to figure out myself what chat app is really respecting "privacy".

**Comment by Fantu** [comment]
Have you looked the matrix app packaged in Debian and derivates? See https://wiki.debian.org/Matrix
You can look also "All known clients" here: https://matrix.org/ecosystem/clients/

**Reply by Jopp-gh** [reply]
The matrix protocol praises itself to be private. So we don't talk about some simple chat app only.

I've been interested in the matrix protocol 2 years ago, informing myself through various documents and legal, official documents too. Chat apps invite to data leakage. We've seen how poor justice is nowadays on the example of whistle blowers. 

You can communicate in two ways, private or in public.

No matter what cheesy stuff I communicate in private chats, private chats need to be private. 
If I talk in public instead, it doesn't matter what chat app I use. 
Of course privacy is a right, not just some gimmick of an app so I check the apps that respect my rights.
